packages/cozy-client/src/CozyClient.spec.js

```diff
@@ -60,6 +60,20 @@ describe('CozyClient initialization', () => {
     client = new CozyClient({ links, schema: SCHEMA })
   })
 
+  it('should autologin when provided token and uri', () => {
+    const token = 'fake_token'
+    const uri = 'https://example.mycozy.cloud'
+    client = new CozyClient({ token, uri })
+    expect(client.isLogged).toBeTruthy()
+  })
+
+  it('should not break explicit login when provided token and uri', () => {
+    const token = 'fake_token'
+    const uri = 'https://example.mycozy.cloud'
+    client = new CozyClient({ token, uri })
+    expect(client.login()).toBeInstanceOf(Promise)
+  })
+
```

Comment:
Can you add a test showing that a second login() returns the promise ?

Reply:
This *is* the second login (the first one is the automatic one). Do you want to test 2 explicit calls to `login`? (it didn't work before my PR so it was not really expected)

Reply:
Ah yes sorry :) All good then ;)